### DISCUSSION TITLE
Allow it to search specific types of content, or address the search to specific sites/repositories

### DISCUSSION THREAD

**Discussion by joarangomo** [qunash/chatgpt-advanced#132] [Ideas]
Hi, I´m an REALLY new begginer in programming, but I´ve been trying to make WebChatGPT able to search for specific sites or even some specific type of result such as articles, journals, papers, repositories, etc. but it has been impossible for me. 

I have tried {web_result:"{query}" site:elsevier.com AND site:scielo.org} as an example, but it still doesn´t work and I don´t know how it can be done :( (this is mostly for educative, investigative purposes)

Thanks in advance! This is a really useful tool and it has made everything a lot easier.

**Comment by qunash** [comment]
Hi,
Search operators like `site:` and others currently do not work in prompt templates. They are query-level, i.e. you should use them within the messages you type into chatgpt.

How this answers your question.
